Validation/Editor/ValidationUtil.cs

```diff
@@ -76,7 +82,19 @@ public static class ValidationUtil {
 			});
 		}
 
-		public static IList<IValidationError> ValidateAllGameObjectsInScenes(IEnumerable<Scene> scenes, bool earlyExitOnError = false) {
+		public static bool HasToPersistScene()
+		{
+			if (UnityEditorInternal.InternalEditorUtility.inBatchMode)
+			{
+				EditorSceneManager.NewScene(NewSceneSetup.EmptyScene, NewSceneMode.Single);
```

Comment:
I get not opening a prompt if in batch mode, but why do we have to create a new scene here? Can you leave a comment?

Reply:
It is done to force the scene to be discarded with all its changes (some plugins/test can modify a scene and this will appear on the DTValidator. Forcing all the changes to be discarded was the solution we found optimal.

Reply:
This doesn't seem like we need to put this in the library given that some users might not expect / want the plugin to discard all changes from the current scene. You could place this modification in a wrapper around the unit test method if you need it in your own project (and/or we could add these variants to this library itself)?

Actually, wouldn't the validator discard the changes anyways once it opens up a new scene? That is why we allow the user to save the current scene, because we will open up a new scene in order to validate it.